cmd/convox/logs.go

```diff
@@ -20,15 +20,15 @@ func init() {
 			rackFlag,
 			cli.StringFlag{
 				Name:  "filter",
-				Usage: "Only return logs that match a filter pattern. If not specified, return all logs.",
+				Usage: "filter the logs by a given token",
 			},
 			cli.BoolTFlag{
 				Name:  "follow",
-				Usage: "Follow log output (default).",
+				Usage: "keep streaming new log output (default)",
```

Comment:
small nitpick but some of the usages start with caps and others don't. the tool would feel more polished with consistency